operator_config.py

```diff
@@ -0,0 +1,25 @@
+OPERATOR_NAME = "heroku"
+
+OPERATOR_MODULE = "heroku-deploy"
```

Comment:
might have to change the name to `heroku_deploy` or just `heroku`. it will be easier for importing the module this way. Module names with `-` can be a bit tricky to import in python.

Reply:
I don't think you can import `-`

Reply:
I guess it works with the `importlib.import_module('heroku-deploy')`? But yeah you can't import it in the python scripts so it's better to change it.

Reply:
I guess we can provide a way to process this, using this https://github.com/pypa/packaging

Reply:
@jjmachan @aarnphm it's better to just follow python conversion and avoid "-", also since this directory is being added to `sys.path`, it can run into conflicts with other packages, e.g. https://pypi.org/project/heroku/. the name `heroku_deploy` seem to be ok https://pypi.org/project/heroku_deploy/

Also since importing this module will mostly be internal(users typically won't be writing python code to import them), we should probably just give all operator modules a common prefix, e.g. `bcdt_heroku_op`?

Reply:
yep adding a prefix is a very good idea. we can start using this thought once the name has been confirmed.

Reply:
I agree. `bcdt_heroku_op` makes more sense and it will be easier to debug than trying to figure out what `heroku-deploy` means. I'll update the name for now, and we can always change it when the name is confirmed (though I think bcdt is a good name)

Reply:
lets just make it `heroku_deploy` for now